tests/v2/test_wakunode.nim

```diff
@@ -1469,10 +1528,10 @@ procSuite "WakuNode":
       # Check that underlying peer info contains only bindIp before starting
       node.switch.peerInfo.addrs.len == 1
       node.switch.peerInfo.addrs.contains(bindEndpoint)
-      
+
       node.announcedAddresses.len == 1
       node.announcedAddresses.contains(announcedEndpoint)
-        
+
```

Comment:
No need to change anything for this PR, but as a general comment: a large number of whitespace-only changes (often caused by prettifying plugins or editors) can make a PR a bit difficult to review and "hide" the changes that are relevant to the issue. Formatting changes are often helpful and necessary, but in future I'd suggest opening a separate PR for formatting only if they become a significant portion of the total changes.

Reply:
Oh, that was unintentional. I agree, that only adds unnecessary noise. I have to disable the "remove trailing spaces" option from the IDE.